app/controllers/about_us_controller.rb

```diff
@@ -0,0 +1,4 @@
+class AboutUsController < ApplicationController
+  def index
+  end
+end
```

Comment:
Why is this controller needed?

Reply:
The way we designed the LO-FI's we have an 'About Us' section we would like to use for the history let's say of the restaurant.
Since this feature is about navigating the site we wrote steps that require the user to click on the 'About Us' link and back.
To have an 'About Us' view then we must have an appropriate controller. Simple one with index only. So as to navigate to the page.